src/validators/base.validator.js

```diff
@@ -9,11 +9,11 @@ module.exports = class BaseValidator {
     pageContext.errors = {}
     pageContext.errorList = []
 
-    // validationErrors.data.details.forEach((error) => {
-    for (let error of validationErrors.data.details) {
-      const fieldName = error.path
+    validationErrors.data.details.forEach((error) => {
+      const errorIndex = error.path.pop() || error.path
```

Comment:
Could do with comments on lines 13 and 14 to explain the purpose, it's not immediately clear from the code.

Reply:
Done